drakrun/drakrun/playground.py

```diff
@@ -7,22 +8,39 @@
 from drakrun.networking import (
     setup_vm_network,
     start_dnsmasq,
+    delete_vm_network,
+    stop_dnsmasq,
 )
 from drakrun.vm import generate_vm_conf, VirtualMachine
 from drakrun.config import InstallInfo, PROFILE_DIR, ETC_DIR
 from drakrun.storage import get_storage_backend
 from drakrun.util import RuntimeInfo, graceful_exit
 from drakrun.injector import Injector
 from drakrun.draksetup import find_default_interface
+import sys
+
+
+def cleanup(vm_id: int):
+
+    logging.info("Ensuring that drakrun@* services are stopped...")
+    try:
+        subprocess.check_output(f'systemctl stop \'drakrun@{vm_id}\'', shell=True, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError:
+        logging.error("drakrun services not stopped")
+        sys.exit(0)
```

Comment:
I think that we can move this function into the `DrakmonShell` class.
Some more observations:
* Message "..ensuring that drakrun@* services..." is wrong - we're stopping only one daemon
* Plase use subprocess.run instead of check_output
* Using argument list will avoid this escaping: `\'drakrun@{vm_id}\''
* Instead of `sys.exit` we should reraise the exception to clean up other resources

Reply:
Done